modules/alphamat/tutorials/alphamat_tutorial.markdown

```diff
@@ -11,9 +11,9 @@ Alphamatting is the problem of extracting the foreground from an image. The extr
 
 Given an input image and its corresponding trimap, we try to extract the foreground from the background. Following is an example:
 
-Input Image: ![](samples/input_images/plant.jpg)
-Input Trimap: ![](samples/trimaps/plant.png)
-Output alpha Matte: ![](samples/output_mattes/plant_result.jpg)
+|Input Image|Input Trimap|Output alpha Matte|
+|:---:|:---:|:---:|
+|![](../samples/input_images/plant.jpg)|![](../samples/trimaps/plant.png)|![](../samples/output_mattes/plant_result.jpg)|
```

Comment:
Please take a look on Doxygen [demo page](http://pullrequest.opencv.org/buildbot/export/pr_contrib/2743/docs/dd/d0e/tutorial_alphamat.html) for this PR.

Avoid using of relative paths, please use `alphamat/samples/...` prefix (`<opencv_contrib>/modules` is added to a search list of base image paths).

Also please revert table layout for this, it is not handled properly by Doxygen.